harness/determined/common/api/experiment.py

```diff
@@ -24,23 +19,20 @@ def patch_experiment(master_url: str, exp_id: int, patch_doc: Dict[str, Any]) ->
     req.patch(master_url, path, json=patch_doc, headers=headers)
 
 
-def activate_experiment(master_url: str, exp_id: int) -> None:
-    path = "/api/v1/experiments/{}/activate".format(exp_id)
-    headers = {"Content-Type": "application/merge-patch+json"}
-    req.post(master_url, path, headers=headers)
-
-
 def follow_experiment_logs(master_url: str, exp_id: int) -> None:
     # Get the ID of this experiment's first trial (i.e., the one with the lowest ID).
     print("Waiting for first trial to begin...")
+    certs.cli_cert = certs.default_load(master_url)
+    authentication.cli_auth = authentication.Authentication(master_url, try_reauth=True)
+    sess = session.Session(master_url, "determined", authentication.cli_auth, certs.cli_cert)
```

Comment:
@rb-determined-ai is there a ticket or anything for unifying how we're using/passing these sessions